release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,44 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-improved-operator-installation-workflow"]
+==== Improved Operator installation workflow with OperatorHub
+
+When administrators install Operators with OperatorHub, they now get immediate
+feedback to ensure that the Operator is installing properly.
+
+[id="ocp-4-6-web-console-improved-operand-details-view"]
+==== Improved Operand details view
+
+You can now see the schema grouping of `specDescriptor` fields and the status of
+your Operands on the Operand's details view, so that you can easily see the
+status and configure the `spec` of the Operand instance.
+
+[id="ocp-4-6-web-console-view-operators-related-objects"]
+==== View cluster Operators' related objects
+
+Previously, when viewing a cluster Operator, it was not clear what resources the
+Operator was associated with. When viewing a resource, it was not apparent that
+that resource might be related or managed by an Operator. When troubleshooting a
```

Comment:
We haven't changed the details page for resources owned by a cluster operator. I'd suggest we take the second sentence "When viewing a resource..." out.